src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/ManageIncentiveReviewsResource.kt

```diff
@@ -0,0 +1,253 @@
+package uk.gov.justice.digital.hmpps.incentivesapi.resource
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.ArraySchema
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.incentivesapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.incentivesapi.dto.CreateIncentiveReviewRequest
+import uk.gov.justice.digital.hmpps.incentivesapi.dto.CurrentIncentiveLevel
+import uk.gov.justice.digital.hmpps.incentivesapi.dto.IncentiveReviewDetail
+import uk.gov.justice.digital.hmpps.incentivesapi.dto.IncentiveReviewSummary
+import uk.gov.justice.digital.hmpps.incentivesapi.service.PrisonerIncentiveReviewService
+import uk.gov.justice.digital.hmpps.incentivesapi.util.ensure
+
+@RestController
+@RequestMapping("/incentive-reviews", produces = [MediaType.APPLICATION_JSON_VALUE])
+@PreAuthorize("hasRole('ROLE_READ_INCENTIVES')")
```

Comment:
As I mentioned [in this other comment](https://github.com/ministryofjustice/hmpps-incentives-api/pull/510/commits/813e741c3d845a38e5405939d1a0cea05a53fccd#r1378675627) I think it may be nicer to be more explicit and call this role `READ_INCENTIVE_REVIEWS` given:
- I think we have freedom here
- it's more explicit/clear